### PR TITLE
[IMP] stock_putaway_product_form: Allow to define rules by product template

### PR DESCRIPTION
**Pull request by sergio-teruel**

### CODE REVIEW

**Review by kevinkhao — APPROVED** [review]
LGTM
Thanks for your help!
Travis is red because there is a minor issue for external_dependencies which is not present in manifest. However pip installs the 2.0.0 version, and 3.0.0 is required. I'm guessing this is why you didn't put it in ?

File: stock_putaway_product_form/models/putaway_strategy.py [hunk 1]
```diff
@@ -1,15 +1,48 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# Copyright 2019 Sergio Teruel - Tecnativa <sergio.teruel@tecnativa.com>
```
Comment (kevinkhao) [inline stock_putaway_product_form/models/putaway_strategy.py]: nitpick: year is not right

File: stock_putaway_product_form/hooks.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
```
Comment (kevinkhao) [inline stock_putaway_product_form/hooks.py]: nitpick: year is not right

**Review by kevinkhao — APPROVED** [review]
LGTM
Thanks for your help!
Travis is red because there is a minor issue for external_dependencies which is not present in manifest. However pip installs the 2.0.0 version, and 3.0.0 is required. I'm guessing this is why you didn't put it in ?

**Comment by sergio-teruel** [comment]
Thanks @kevinkhao changes done!!

**Pull request merged**